docs/dashboard/install-guide/quickStart.md

```diff
@@ -46,13 +63,20 @@ curl -s https://api.github.com/repos/NationalSecurityAgency/skills-service/relea
 Using the downloaded jar you can run the SkillTree dashboard and service (make sure to substitute ``X.X.X`` for an actual version):
 
 ```bash
-$ java -jar ~/Downloads/skills-service-X.X.X.jar
+$ java -Dspring.datasource.url=jdbc:postgresql://localhost:5432/skills \
+-Dspring.datasource.username=skills \
+-Dspring.datasource.password=skillsPassword \
+-jar ~/Downloads/skills-service-X.X.X.jar
 ```
 
 <import-content path="/dashboard/install-guide/common/service-install-output-and-backend.html"/>
 
 ## 3. Integrate Client Libraries
 
+::: tip Please Note
+Please note that the integrating the SkillTree Client Libraries is **optional** and is only applicable when integrating gamification training directly into an existing or new web application.
```

Comment:
Should probably be `Please note that integrating`

Reply:
:+1: